backend/events/apps.py

```diff
@@ -85,6 +85,24 @@ def on_tuples(cc_event, block_number, tx_id, tx_status):
                 queue=worker_queue
             )
 
+        if 'ComputePlans' in payload:
```

Comment:
I think you have to handle the tuple/compute plan differentiation before that. At [L51](https://github.com/SubstraFoundation/substra-backend/blob/14804e7fd2ff97ad2ac2599bed5bcb4ce6a01baa/backend/events/apps.py#L51) we have no means yet to differentiate between tuple type lists and the compute plan. So the compute plan will be treated as a tuple and I think it will failed somewhere after that.

Reply:
Good catch!

Reply:
Fixed: https://github.com/SubstraFoundation/substra-backend/pull/90/commits/c29f41d927b2b1db1a802818f349a47d976714cf